sdk/identity/azure-identity/tests/test_shared_cache_credential.py

```diff
@@ -399,9 +388,7 @@ def test_same_tenant_different_usernames():
         credential.get_token("scope")
     # error message should indicate multiple matching accounts, and list discovered accounts
```

Comment:
This comment should be updated or removed because we no longer expect the message to list discovered accounts.